### DISCUSSION TITLE
downloading snapshot: how to change the resolution ?

### DISCUSSION THREAD

**Discussion by vincentberenz** [danielfernau/unifi-protect-video-downloader#455] [General]
Thanks for making this available to us !

When taking and downloading a snapshot using protect-archiver, its resolution is 640 x 360.

When taking a snapshot using the protect web interface, the resolution is 3840 x 2160 (I did set "High Resolution" in protect / Settings / Advanced / Real Time Streaming Protocol).

Would there be a way for protect-archiver to request a higher resolution, or to use the protect active configuration ?

**Comment by danielfernau** [comment]
Interesting idea, I'd have to look into this in a bit more detail before confirming it - but if the web interface can do it, the tool should be able to replicate the request.

A re-write of the tool is in progress, so stay tuned!
